feature-libs/my-account/organization/core/utils/serializer.ts

```diff
@@ -0,0 +1,85 @@
+import { EntitiesModel, ListModel, StateUtils } from '@spartacus/core';
+import { B2BSearchConfig } from '../model/search-config';
+import { Management } from '../store/organization-state';
+
+// TODO after update typescript to 3.7 it can be replaced by Nullish Coalescing (??) operator
+function nullish(param, defaultValue) {
+  return param !== null && param !== undefined ? param : defaultValue;
+}
+
+export const ALL = 'all';
+
+export function serializeB2BSearchConfig(
```

Comment:
IMO serialization/deserialization of the listing data and keeping only list of IDs per `page+pageSize+sort` is a nice, performant concept that may be useful also in other libs. To the extent that maybe some other params will take part.
I feel sooner or later we will need to turn it into more generic functions exposed in the core lib. But I'm happy to leave it as is for now (as it's not exported in the public api)

Reply:
@Platonn you might want to create a ticket out of this

Reply:
https://github.com/SAP/spartacus/issues/9045
@dunqan @marlass FYI